assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/codenvy/che.properties

```diff
@@ -418,12 +418,30 @@ che.infra.openshift.bootstrapper.server_check_period_sec=3
 # Defines whether use the Persistent Volume Claim for che workspace needs
 # e.g backup projects, logs etc or disable it.
 che.infra.openshift.pvc.enabled=true
-# Defines the name of Persistent Volume Claim for che workspace.
+
+# Defined which strategies will be used while choosing PVC for workspaces.
+#
+# Supported strategies:
+# - 'onePerProject'
+#        PVC with name that is configured by 'che.infra.openshift.pvc.name'
+#        the property will be used for storing workspaces data of each workspace
+#        in an OpenShift project.
+#        Existing PVC will be used or new one will be created if it doesn't exist.
+#
+# - 'onePerWorkspace'
+#        PVC with name that is evaluated as '{che.infra.openshift.pvc.name} + {WORKSPACE_ID}'
+#        will be used for storing of workspaces data.
+#        Existing PVC will be used or a new one will be created if it doesn't exist.
+che.infra.openshift.pvc.strategy=onePerProject
```

Comment:
question : could it be only "project" and "workspace" (easier to type, no need to be wrong about if it's camelCase or not, etc)

but, it's strange for me to call it with "perProject/project" as I'm thinking that each project of a workspace will have its own PVC while it's more for "all workspaces"

I would more use "unique" or "single" than "per project" to me

With "workspace" it's pretty straightforward to understand that it will be one PVC per workspace

Reply:
@benoitf 
I'm ok with using small significant names for such things, so changed supported values to:
`common` - onePerProject
`unique` - onePerWorkspace

Reply:
ok, maybe shared was more meaningful than "common" but I'm fine as well.